app/models/tradingpremises/TradingPremises.scala

```diff
@@ -65,12 +65,13 @@ case class TradingPremises(
     this.copy(whatDoesYourBusinessDoAtThisAddress = Some(p), hasChanged = hasChanged || !this.whatDoesYourBusinessDoAtThisAddress.contains(p),
       hasAccepted = hasAccepted && this.whatDoesYourBusinessDoAtThisAddress.contains(p))
 
-  def msbServices(p: MsbServices): TradingPremises =
+  def msbServices(p: models.tradingpremises.TradingPremisesMsbServices): models.tradingpremises.TradingPremises =
```

Comment:
Are we able to shorten/remove the package name on these types, now that it's clear we're dealing with Trading Premises and Trading Premises MSB Services?

Reply:
Done